tensorflow/core/common_runtime/gpu/gpu_id.h

```diff
@@ -25,10 +25,10 @@ namespace tensorflow {
 //   physical machine, it can be filtered by CUDA environment variable
 //   CUDA_VISIBLE_DEVICES. Note that this id is not visible to Tensorflow, but
 //   result after filtering by CUDA_VISIBLE_DEVICES is visible to TF and is
-//   called CUDA GPU id as below. See
+//   called platform GPU id as below. See
```

Comment:
Currently this tightly coupled with CUDA: `...it can be filtered by CUDA environment variable CUDA_VISIBLE_DEVICES. Note that this id is not visible to Tensorflow, but result after filtering by CUDA_VISIBLE_DEVICES is visible to TF...`.

Any plan to change that as well? Or does ROCm has similar environment variable for same functionality?